src/test/java/seedu/address/logic/commands/ListAbsenteesCommandTest.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.FIONA;
+import static seedu.address.testutil.TypicalPersons.GEORGE;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.DateAbsentPredicate;
+
+public class ListAbsenteesCommandTest {
+    private static final LocalDateTime FIRST_DATE_TIME = LocalDateTime.of(2024, 1, 1, 12, 0);
+    private static final LocalDateTime SECOND_DATE_TIME = LocalDateTime.of(2024, 1, 8, 12, 0);
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        DateAbsentPredicate firstPredicate = new DateAbsentPredicate(FIRST_DATE_TIME);
+        DateAbsentPredicate secondPredicate = new DateAbsentPredicate(SECOND_DATE_TIME);
+
+        ListAbsenteesCommand firstCommand = new ListAbsenteesCommand(firstPredicate);
+        ListAbsenteesCommand secondCommand = new ListAbsenteesCommand(secondPredicate);
+
+        // same object -> returns true
+        assertTrue(firstCommand.equals(firstCommand));
+
+        // same values -> returns true
+        ListAbsenteesCommand firstCommandCopy = new ListAbsenteesCommand(firstPredicate);
+        assertTrue(firstCommand.equals(firstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(firstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(firstCommand.equals(null));
+
+        // different dates -> returns false
+        assertFalse(firstCommand.equals(secondCommand));
+    }
+
+    @Test
+    public void execute_validKeyword_success() {
```

Comment:
Does this test name accurately describe the test?

Reply:
Good point. The main goal for this test is to check whether the command returns the expected results given a valid keyword; the keyword itself should not be the emphasis.

Would a name like `execute_validKeyword_returnsCorrectly` be okay?

Reply:
I feel like using the term `date` rather than `keyword` is more accurate to describe the parameter that is being passed into the function.